src/main/scala/edu/ie3/mobsim/io/geodata/PointOfInterest.scala

```diff
@@ -43,6 +46,7 @@ import scala.jdk.CollectionConverters._
 final case class PointOfInterest(
     uuid: UUID,
     id: String,
+    poiType: PoiTypeDictionary.Value,
```

Comment:
Since `poiType` can be completely derived from `categoricalLocation`, once could also just provide a method here instead of saving it as a field.